shimming-toolbox/shimmingtoolbox/optimizer/lsq_optimizer.py

```diff
@@ -190,21 +190,9 @@ def _residuals_mse(self, coef, a, b, c):
         # The first version was :
         # np.mean((unshimmed_vec + np.sum(coil_mat * coef, axis=1, keepdims=False))**2) / factor + \ (
         # self.reg_factor * np.mean(np.abs(coef) / self.reg_factor_channel))
-        # For the second version we switched np.sum(coil_mat*coef,axis=1,keepdims=False) by coil_mat@coef
-        # which is way faster Then for a vector, mean(x**2) is equivalent to x.dot( x)/n
-        # it's faster to do this operation instead of a np.mean Finally np.abs(coef).dot(self.reg_vector) is
-        # equivalent and faster to self.reg_factor*np.mean(np.abs(coef) / self.reg_factor_channel) For the
-        # mathematical demonstration see : https://github.com/shimming-toolbox/shimming-toolbox/pull/432
-        # This gave us the following expression for the residuals mse :
-        # shimmed_vec = unshimmed_vec + coil_mat @ coef
-        # mse = (shimmed_vec).dot(shimmed_vec) / len(unshimmed_vec) / factor + np.abs(coef).dot(self.reg_vector)
```

Comment:
This seems very useful to keep track of the changes that were made over the years.